src/OS.c

```diff
@@ -6,104 +6,119 @@
  */
 
 #include "OS.h"
+
 #include "tm_stm32_gps.h"
 #include "GSM.h"
 #include "ESP01.h"
 #include "MPU6050.h"
 #include "MQTT_Tasks.h"
 #include "socket_Task.h"
+
+#include "MPU6050.h"
 #include "Accident.h"
+#include "task.h"
+
+#include "queue.h"
+#include "semphr.h"
+#include "stm32f4_discovery.h"
 /*create all tasks here*/
 
 
 extern TaskHandle_t handle_gsm_os_init;
 
-extern TaskHandle_t xTask_MPU6050_Read_RawData;
+TaskHandle_t xTask_MPU6050_Read_RawData;
+
+TaskHandle_t xTask_AccidentDetection_Declration;
+
+xQueueHandle Accident_Detection_Queue;
+xSemaphoreHandle Accident_Detection_semphr;
+
```

Comment:
create all handlers in it's own module and only extern them here to avoid fatal errors when removing a task from the OS

Reply:
it worked properly there is  no fatal errors  , stop making things up

Reply:
ok :''D